src/main/java/com/limechain/cli/Cli.java

```diff
@@ -49,8 +48,9 @@ public CliArguments parseArgs(String[] args) {
             String network = cmd.getOptionValue("network", "").toLowerCase();
             if (!validChains.contains(network) && !network.isEmpty()) throw new ParseException("Invalid network");
             String dbPath = cmd.getOptionValue("db-path", DBInitializer.DEFAULT_DIRECTORY);
+            boolean dbRecreate = cmd.hasOption(DB_RECREATE);
```

Comment:
Not related to the implementation but I think it might be a good idea to extract the other cli options - `network` and `db-path` into static final variables just to keep the code tidy and organized.